plugins/nirmata/plugin.go

```diff
@@ -0,0 +1,22 @@
+package nirmata
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema"
+)
+
+func New() schema.Plugin {
+	return schema.Plugin{
+		Name: "nirmata",
+		Platform: schema.PlatformInfo{
+			Name:     "Nirmata",
+			Homepage: sdk.URL("https://nirmata.com"), // TODO: Check if this is correct
```

Comment:
Let's remove the TODO from here.

Reply:
yess into it

Reply:
@itsCheithanya This does not seem to be removed yet, could we make sure to remove it, please?